05/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>devmentor.pl - HTML & CSS: RWD - #05</title>
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/global.css" />
+    <link
+      rel="stylesheet"
+      media="(max-width: 799px)"
+      href="./styles/mobile.css"
+    />
+    <link
+      rel="stylesheet"
+      media="(min-width: 800px)"
+      href="./styles/desktop.css"
+    />
+  </head>
+  <body>
+    <header class="header row">
+      <div class="header__container container">
+        <div class="header__logo cell"></div>
+        <label for="btn" class="header__burger burger__btn">
```

Comment:
Jeśli mamy element BEM tj. `burger__btn` to w hierarchii powinni gdzieś być block BEM tj. `burger` tutaj tego brakuje.
Osobiście tutaj wstawiłbym sam block i byłoby ok tj. `burger`